### DISCUSSION TITLE
cant get it to work

### DISCUSSION THREAD

**Discussion by Volanaro** [Etaash-mathamsetty/trigger-control#22] [Q&A]
hi im following the steps and i just cant get it to work on arch im getting this error
`-- Check for working C compiler: /usr/bin/cc - broken
CMake Error at /usr/share/cmake/Modules/CMakeTestCCompiler.cmake:69 (message):
  The C compiler

    "/usr/bin/cc"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /home/deck/trigger-control/build/CMakeFiles/CMakeTmp
    
    Run Build Command(s):/usr/bin/make -f Makefile cmTC_0c45c/fast && /usr/bin/make  -f CMakeFiles/cmTC_0c45c.dir/build.make CMakeFiles/cmTC_0c45c.dir/build
    make[1]: Entering directory '/home/deck/trigger-control/build/CMakeFiles/CMakeTmp'
    Building C object CMakeFiles/cmTC_0c45c.dir/testCCompiler.c.o
    /usr/bin/cc    -o CMakeFiles/cmTC_0c45c.dir/testCCompiler.c.o -c /home/deck/trigger-control/build/CMakeFiles/CMakeTmp/testCCompiler.c
    /usr/lib/gcc/x86_64-pc-linux-gnu/12.2.0/cc1: error while loading shared libraries: libisl.so.23: cannot open shared object file: No such file or directory
    make[1]: *** [CMakeFiles/cmTC_0c45c.dir/build.make:78: CMakeFiles/cmTC_0c45c.dir/testCCompiler.c.o] Error 1
    make[1]: Leaving directory '/home/deck/trigger-control/build/CMakeFiles/CMakeTmp'
    make: *** [Makefile:127: cmTC_0c45c/fast] Error 2
    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:4 (project)

-- Configuring incomplete, errors occurred!
See also "/home/deck/trigger-control/build/CMakeFiles/CMakeOutput.log".
See also "/home/deck/trigger-control/build/CMakeFiles/CMakeError.log".
make: *** No targets specified and no makefile found.  Stop.
[sudo] password for deck: 
cp: cannot stat 'trigger-control': No such file or directory
`

**Comment by BananaWorks07** [comment]
use the app image build from the releases
based on your username it looks like you are on the steam deck which doesn't have a working compiler ootb
